components/raftstore-v2/src/raft/peer.rs

```diff
@@ -84,6 +84,7 @@ pub struct Peer<EK: KvEngine, ER: RaftEngine> {
     /// Transaction extensions related to this peer.
     txn_ext: Arc<TxnExt>,
     txn_extra_op: Arc<AtomicCell<TxnExtraOp>>,
+    pending_ticks: Vec<PeerTick>,
```

Comment:
Is it more appropriate not to put it under the "transaction extentions" section? Most ticks don't seem to be related to transactions.